components/ws-manager-bridge/src/prometheus-metrics-exporter.ts

```diff
@@ -26,6 +29,16 @@ export class PrometheusMetricsExporter {
             labelNames: ['region'],
             buckets: prom.exponentialBuckets(2, 2, 10),
         });
+        this.clusterScore = new prom.Gauge({
+            name: 'gitpod_ws_manager_bridge_cluster_score',
+            help: 'Score of the individual registered workspace cluster',
+            labelNames: ["cluster"]
```

Comment:
There is no need to add the cluster label. Prometheus already adds this label to all metrics when it remote-writes to a Long Term Storage.

If we keep it, we'll just have duplicated labels

Reply:
If you really want to keep this label. I'd ask to at least change the label name... On monitoring-satellite configuration we've already made `cluster` sort of a "reserved" label that is applied to every metric 😬

Reply:
This is not the same cluster as the one this metric originates from. Instead, it's the registerd workspace cluster for which we're reporting the score.
I'll change this to `workspace_cluster`.

Reply:
Hmmm thanks for the clarification! For some reason I thought ws-manager-bridge was running in the workspace clusters 🤦‍♂️.

This means that my implementation in https://github.com/gitpod-io/gitpod/pull/5557 was completely wrong